### PR TITLE
Isotope pattern plot

### PR DESCRIPTION
**Pull request by lomereiter**
Computes prof_mzs/prof_ints instead of loading them from DB.

Also I switched to `plotly.js` because it's more familiar to me than `amCharts`.

### CODE REVIEW

File: sm/webapp/static/js/chart.js [hunk 1]
```diff
@@ -1,112 +1,58 @@
+function centroidPlot(data) {
+    var mzs = data['mzs'];
+    var ints = data['ints'];
+    var mzs2 = [];
+    var ints2 = [];
+    for (var i = 0; i < mzs.length; i++) {
+        mzs2.push(mzs[i] - 1e-6);
+        mzs2.push(mzs[i]);
+        mzs2.push(mzs[i] + 1e-6);
```
Comment (intsco) [inline sm/webapp/static/js/chart.js]: What is the purpose of subtracting/adding 1e-6 to all centroid mzs?

Comment (lomereiter) [inline sm/webapp/static/js/chart.js]: It's a horrible hack to have (almost) vertical lines as centroids. I tried to use bar plots first, but there are no options to specify bar width yet, so I resorted to a line plot.

File: sm/webapp/static/js/chart.js [hunk 2]
```diff
@@ -1,112 +1,58 @@
...
+        var layout = {
+            xaxis: {'range': [min_mz, max_mz]},
+            yaxis: {title: 'Intensity (a. u.)', rangemode: 'nonnegative'},
+            legend: {x: 0.5, y: -0.2, xanchor: 'center', yanchor: 'top',
+                     orientation: 'h', traceorder: 'reversed'},
+            margin: {t: 20, b: 20},
+            font: {size: 16}
+        };
+
+        var sampleData = centroidPlot(data['sample'])
```
Comment (intsco) [inline sm/webapp/static/js/chart.js]: lomereiter, are you using ES6 style with no semicolons?

Comment (lomereiter) [inline sm/webapp/static/js/chart.js]: Not yet, I'd rather avoid discussions like https://www.reddit.com/r/programming/comments/sag8p/crockford_on_bootstraps_semicolon_omission/

**Pull request merged**